build/configs/artik053/extra/defconfig

```diff
@@ -844,7 +907,7 @@ CONFIG_FS_PROCFS=y
 # CONFIG_FS_PROCFS_EXCLUDE_PARTITIONS is not set
```

Comment:
enable proc

Reply:
This config is `EXCLUDE` feature. Should I enable this?